packages/editor/src/functions/addMediaNode.ts

```diff
@@ -69,6 +68,12 @@ export async function addMediaNode(
   const contentType = (await getContentType(url)) || ''
   const { hostname } = new URL(url)
 
+  const pathArray = url.split('/')
+  const lastIndex = pathArray.length - 1
+  const fileNameWithExt = pathArray[lastIndex]
+  const fileNameArray = fileNameWithExt.split('.')
+  const name = decodeURI(fileNameArray[0])
```

Comment:
We should wrap the logic here in a try catch as there's a chance the url comes in with invalid characters for decodeURI or somehow the